azure-pipelines.yml

```
@@ -216,6 +216,16 @@ jobs:
         eval "$(./dev-env/bin/dade-assist)"
         mkdir -p $(Build.StagingDirectory)/split-release
         ./ci/assembly-split-release-artifacts.sh $(release_tag) $(Build.StagingDirectory)/release-artifacts $(Build.StagingDirectory)/split-release
+    - bash: |
+        set -euo pipefail
+        source $(bash-lib)
+        cd $(Build.StagingDirectory)/split-release/split-release
+        for f in "damlc-*" daml-libs/daml-script; do
```

```suggestion
        for f in "damlc-*" daml-libs/daml-script daml-libs/daml-trigger; do
```
I‘m also not sure listing those in here is a good idea. The list of libraries seems like the type of thing that is going to change across releases and blow up once we do releases for an older branch. Should we move this to a separate bash script?

---

Can't imagine that blowing up unless we delete `daml-script`. Library dependencies seem largely additive, so worst case we push more than needed for a patch release.

@oliverse-da told me they're not using `daml-trigger`. Note that this is intended as a convenience for local development; all the artifacts are still on Artifactory and I would still expect CI to get them all from there.

---

What I’m worried about is that we add a new lib in the future that we want to use in Canton and then it blows up on older releases. But if you’re comfortable addressing it at that point, I’m fine with it for now.

---

I don't think this is likely enough to be worth a separate file here, partly because we're copying from a folder that is itself already created by a separate file, so we have an easy way to fix it should it happen.